hypersistence-optimizer-spring-boot-example/pom.xml

```diff
@@ -32,7 +32,8 @@
     </dependencies>
 
     <properties>
-        <spring-boot.version>2.1.3.RELEASE</spring-boot.version>
+        <spring-boot.version>2.4.1</spring-boot.version>
+        <hikari.version>3.4.5</hikari.version>
```

Comment:
Hikari version specified in parent pom.xml is incompatible with latest spring-boot release, so hikari version is explicitly mentioned which is compatible with spring-boot-2.4.1.